src/components/Map.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import France from "@svg-maps/france.regions";
+import {SVGMap } from "react-svg-map"; 
+import "react-svg-map/lib/index.css";
+ 
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Your component is only presentational, it does not have state nor any logic. Use a function instead.